tests/Traits/ValidCodeAnalysisTestTrait.php

```diff
@@ -70,21 +70,29 @@ public function testValidCode(
         $this->analyzeFile($file_path, $context);
 
         $actual_vars = [];
-        foreach ($assertions as $var => $_) {
+        foreach ($assertions as $expr => $_) {
             $exact = false;
 
+            $var = $expr;
```

Comment:
Is there any reason to alias this? `$expr` doesn't seem to be used in the loop body.

Reply:
Nope, that must be a leftover change from when I was figuring out how to get it working.